processes/presenting-at-conferences.rst

```diff
@@ -0,0 +1,154 @@
+#########################
+Presenting at Conferences
+#########################
+
+This is our playbook for presenting on behalf of DM at both project and public meetings.
+
+.. seealso::
+
+   Conference proceedings papers are coordinated by the LSST Project Publication Board.
+   See :doc:`publication-policy` for more information.
+
+.. _presenting-interaction-report:
+
+Interaction report
+==================
+
+As DM team members, we work within an NSF and DOE funded MREFC construction project.
+It's important that we're consistent with the Project when we speak in public and with government officials.
+As you're preparing your presentation:
+
+- `File an interaction report <https://project.lsst.org/interactions>`__. Interaction reports are required when presenting at a public meeting or with government officials.
```

Comment:
I have tried on multiple occasions to "do the right thing" but it seems like a lot of these processes were put in place pre-construction and have not been updated with current expectations. I've definitely been told that no-one is looking at the submissions even when it worked but it's senior management level who have to decide whether to drop it completely or not.